web/test/pages/dashboard.test.tsx

```diff
@@ -93,6 +95,14 @@ describe("dashboard", () => {
     expect(subject.getByText("Annual Report")).toBeInTheDocument();
   });
 
+  it("displays filings calendar without annual report date if date of formation undefined", () => {
+    const userData = generateUserData({ profileData: generateProfileData({ dateOfFormation: undefined }) });
+    useMockUserData(userData);
+
+    const subject = renderPage({});
+    expect(subject.queryByText("Annual Report")).not.toBeInTheDocument();
+  });
+
```

Comment:
I don't think this test is necessary / actually testing this logic - I believe it would pass even if date of formation is required, right? Because what shows in the calendar is controlled by what is in the user's `filings` data, which is computed on the backend using their date of formation. But for a frontend test here, I can't imagine a situation in which this test would ever fail. So I don't think we need to have it.

Reply:
Ohh I see I didn't realize it connects tot the backend to provide the filings from date. Makes sense, I'll remove this